src/content/en/tools/chrome-devtools/_shared/_feedback.html

```diff
@@ -0,0 +1,41 @@
+{% framebox width="auto" height="auto" %}
```

Comment:
In order for widgets to work inside a framebox, the first line needs to be:
`{% framebox width="auto" height="auto" enable_widgets="true" %}`